run_benchmarks.sh

```diff
@@ -0,0 +1,28 @@
+TMPDIR="${PWD}/bundles"
+if [ -d "$TMPDIR" ]; then
+  # Take action if $DIR exists. #
+  echo "removing ${TMPDIR} before running benchmarks"
+  rm -rf ${TMPDIR}
+fi
+mkdir ${TMPDIR}
```

Comment:
@milkysoft PWD is a repository location, for temp directories you can use `TMPDIR=$(mktemp --directory)`

Reply:
@g4s8 , fixed with latest [commit](https://github.com/artipie/rpm-adapter/pull/386/commits/7607ca7c554416dcdf9bcb7723af6fc9e1f3c09b)